stats/opentelemetry/server_metrics.go

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package opentelemetry
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/internal"
+	"google.golang.org/grpc/stats"
+	"google.golang.org/grpc/status"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+type serverStatsHandler struct {
+	mo MetricsOptions
+
+	serverMetrics serverMetrics
+}
+
+func (ssh *serverStatsHandler) initializeMetrics() {
+	// Will set no metrics to record, logically making this stats handler a
+	// no-op.
+	if ssh.mo.MeterProvider == nil {
+		return
+	}
+
+	meter := ssh.mo.MeterProvider.Meter("gRPC-Go")
+	if meter == nil {
+		return
+	}
+	setOfMetrics := ssh.mo.Metrics.metrics
+
+	serverMetrics := serverMetrics{}
+	if _, ok := setOfMetrics["grpc.server.call.started"]; ok {
+		scs, err := meter.Int64Counter("grpc.server.call.started", metric.WithUnit("call"), metric.WithDescription("The total number of RPCs started, including those that have not completed."))
```

Comment:
Description - "Number of server calls started"
Unit - "{call}"

Reply:
Done.